parser/quakec-parser.js

```diff
@@ -824,7 +839,7 @@ class Parse {
         let statement;
 
         while(true) {
-            if (Context.token.id === "}") {
+            if (Context.token.id === "}" || Context.token.id === "(end)") {
```

Comment:
Stop parsing on end. :)

Reply:
I also made this change in my stability branch. 😄